apps/valkyrie/rel/plugins/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.*
```

Comment:
You should be able to remove the entire `rel` directory from valkyrie to follow the same patterns andi/forklift do.